### ISSUE TITLE
Verify focus behavior

### ISSUE THREAD

**Issue by johnpryan** [dart-lang/dart-pad#2217]
This is an issue to track any focus management issues. Some users have reported that focus can get stuck outside of the Flutter panel, especially when switching between steps in a workshop.

**Comment by miquelbeltran** [comment]
I am currently reviewing some DartPad based workshops and got stuck on this issue, I've found a minimal reproducible example:

1. Open https://dartpad.dev/workshops.html?webserver=https://raw.githubusercontent.com/flutter/codelabs/master/dartpad_codelabs/src/custom_button#Step10
3. Click Run
2. Click on the empty white space in the app in the UI Output
4. You can press `tab` to select the button (changes color) and `space` to click it, some messages appear in the Console.
5. Navigate to Step 9
6. Navigate back to Step 10
7. Click Run

Now, the user is unable to get focus on the Button. Only a full page refresh (cmd + shift + R).

I can confirm this happening both with Firefox and Chrome, as well as in Linux and MacOS.

**Comment by johnpryan** [comment]
I think this will be fixed by https://github.com/dart-lang/dart-pad/pull/2270, but need to verify

**Comment by johnpryan** [comment]
fixed by #2270
